src/is/types.js

```diff
@@ -158,6 +159,19 @@ export function isWindowObject(value) {
     return value != null && typeof value === 'object' && 'setInterval' in value;
 }
 
+/**
+ * Determines whether a given value is a uri
+ * @param {*} value the value to check
+ * @returns {Boolean} `true` if the value is a URI; otherwise, `false`
+ */
+export function isURI(value) {
```

Comment:
Shouldn't be in this file, but rather `regexp.js` as mentioned in the issue description.

Reply:
got it